src/uu/cut/src/whitespace_searcher.rs

```diff
@@ -0,0 +1,99 @@
+// This file is part of the uutils coreutils package.
+//
+// (c) Rolf Morel <rolfmorel@gmail.com>
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+// cSpell:ignore multispace
```

Comment:
Nit: we usually write `spell-checker:ignore` (but the effect is the same)

Reply:
done